s3.tf

```diff
@@ -1,7 +1,33 @@
 resource "aws_s3_bucket" "demo-example" {
```

Comment:
**Accurics remediation**
 
**[HIGH]: S3 bucket accessible to all authenticated users.**
 
**Vulnerability**
S3 bucket Access is allowed to all AWS Account Users.
 
**Remediation**
S3 bucket Access should be limited to specific users and roles.

source: accurics-